addons/website_event_questions/static/src/js/many2many_tags_answers.js

```diff
@@ -0,0 +1,23 @@
+odoo.define('website_event_questions.many2many_tags_answers', function (require) {
+    "use strict";
+
+    var FieldMany2ManyTags = require('web.relational_fields').FieldMany2ManyTags;
+    var registry = require('web.field_registry');
+
+    var FieldMany2ManyTagsAnswers = FieldMany2ManyTags.extend({
+        className: "o_field_many2manytags o_field_many2manytags_answers",
+        /**
+         * @override
+         */
+        _getRenderTagsContext: function () {
+            var res = this._super.apply(this, arguments);
+            res['elements'].forEach(el => {
+                if (!el['display_name']) {
+                    el['display_name'] = el['name'];
+                }
```

Comment:
I create the widget because when you add a question and answers, the display name is computed when you click on the button save on the event form. So with many2many_tags, when you create a question and answers, it displays a tag with nothing inside, since display_name is not set.